device-types/QNAP/TS-563.yml

```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: QNAP
+model: TS-563
+slug: qnap-ts-563
+part_number: TS-563
+u_height: 2
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (9.7 lbs or 4.4 kg)